001-core/privacy.php

```diff
@@ -457,3 +457,22 @@ function delete_old_export_files() {
 		}
 	}
 }
+
+/**
+ * Disable crawling for go-vip.co and go-vip.net domains.
+ * 
+ * @param string $output The robots.txt content.
+ * @return string The modified robots.txt content.
+ */
+function vip_convenience_domain_robots_txt( $output ) {
+	$host = strtolower( $_SERVER['HTTP_HOST'] ?? '' ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+	if ( false !== strpos( $host, '.go-vip.co' ) || false !== strpos( $host, '.go-vip.net' ) ) {
```

Comment:
Should these check for the strings at the _end_ of the `$host`, rather than just the presence of it? `develop.go-vip.company.com` would satisfy this condition, for instance.

Does any IPv4 or IPv6 need to be taken into account as potential values of `$host`?